src/main/main.ts

```diff
@@ -27,9 +27,12 @@ import { getInstalledActivities } from '../utils/getInstalledActivities'
 import { runCommand } from '../utils/runCommand'
 
 // Auto update
-require('update-electron-app')({
-  repo: 'will-stone/browserosaurus',
-})
+// disables if no apple developer id for signing app
+if (process.env.APPLE_ID) {
```

Comment:
@chrisands Are you getting updates for 6.5.1? I think this hasn't worked.

Reply:
I'm updating via `brew` so I don't have this problem, but I don't have any autoupdate prompt.

Reply:
Okay, cool. I've pushed out 6.5.2 to fix the issue.